SSDReco/SSDHotChannelList.cxx

```diff
@@ -136,9 +136,9 @@ namespace emph {
 	  if (diff/sigFinal < signif) continue; // O.K.  
 	  fHotChannels.push_back(k+3);
 	} // last channel, who knows.. 
+        std::cerr << " SSDHotChannelList::tallyIt, for Station " << fStation << " " << fSensor << " num Hot " << fHotChannels.size() << std::endl;
```

Comment:
i'd prefer an MFLogError here though recognise it's not your implementation. it's not critical